src/Administration/Resources/app/administration/src/module/sw-product/page/sw-product-detail/index.js

```diff
@@ -498,6 +498,19 @@ Component.register('sw-product-detail', {
                     }
 
                     default: {
+                        if (response.response.data.errors[0].code === 'CONTENT__DUPLICATE_PRODUCT_NUMBER') {
```

Comment:
Can you make this if condition more save?

Example: If `errros` is missing in the Response if will throw here an JS error

Reply:
Done!